### ISSUE TITLE
Warn when given an unknown ruleset tag or rule ID 

### ISSUE THREAD

**Issue by jasonkarns** [dequelabs/axe-core#82]
```js
a11yCheck({}, {
  runOnly: {
    type: "tag",
    values: ["my-made-up-tag"]
  }
})
```

I would expect this to throw an error for `my-made-up-tag` not being a known ruleset tag. However, it successfully returns 0 violations. (Presumably because the tag doesn't exist, so the rule-set that was checked is empty, thus finding 0 violations.)

Additionally:
```js
a11yCheck({}, {
  runOnly: {
    type: "rule",
    values: ["fakeRule"]
  }
})
```

also successfully returns 0 violations, rather than an error even though `fakeRule` doesn't exist.

I would expect both of these scenarios (as well as with `{rules: {fakeRule: {enabled:true}}}`) to return errors stating that the rule ID or tag doesn't exist.

**Comment by dylanb** [comment]
depends on https://github.com/dequelabs/axe-core/issues/83

**Comment by WilcoFiers** [comment]
We currently have an implementation in our private repo which throws an error if this occurs. That wouldn't require the error first cb implementation.
